.github/workflows/cypress.yml

```diff
@@ -9,4 +9,10 @@ jobs:
       - name: Cypress run
         uses: cypress-io/github-action@v1
         with:
+          build: npm run build
           start: npm start
+          wait-on: http://localhost:5000
+          record: true
+        env:
+          # pass the Dashboard record key as an environment variable
+          CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
```

Comment:
Thank you for the comment! 😄

Reply:
Sure thing!







_(This was copy pasted from Cypress's GH Action Docs)_